ldap/client.go

```diff
@@ -56,6 +59,26 @@ func NewClient(bindUsername, bindPassword, ldapServerUrl string) Client {
 
 func (c *ldapClient) Connect() error {
 	ldapConn, err := ldap.Dial("tcp", c.ldapServerUrl)
+	logger.Debugf("%v got URL as ", c.ldapServerUrl)
+	if err != nil {
+		return fmt.Errorf("Cannot connect to LDAP: %v", err)
+	}
+
+	err = ldapConn.Bind(c.bindUsername, c.bindPassword)
+	if err != nil {
+		ldapConn.Close()
+		return fmt.Errorf("Cannot bind to LDAP: %v", err)
+	}
+
+	c.ldapSearcher = ldapConn
+
+	return nil
+}
+
+func (c *ldapClient) ConnectTls() error {
+	var config tls.Config
+	config.InsecureSkipVerify = true
```

Comment:
It might be better if `InsecureSkipVerify` is something the user can configure, rather than it being hardcoded to `true` - could we add a configuration option to the pack that defaults to `false`?

Reply:
I have added the environment variable , default value shall be false.